spec/models/project_spec.rb

```diff
@@ -257,12 +260,14 @@
 
           expect(project.status).to eq("Deprecated")
           expect(project.deprecation_reason).not_to eq(nil)
+          expect(project.status_checked_at).to eq(Time.current)
+          expect(project.updated_at).to eq(Time.current)
```

Comment:
this Time.current use will be a flaky test - could check that it's within the last few seconds maybe, or use a solution like https://medium.com/@tonia.saba/how-to-freeze-time-in-rspec-tests-7a4888c31f47 if you prefer

Reply:
Will it be flaky with the `travel_to` [here](https://github.com/librariesio/libraries.io/pull/3112/files#diff-1ed876d42b4d51d3aaa37d125efd84dd5f978593126d445c905d1f415c9b6828R250)? My understanding was the `travel_to` would prevent flakiness issues like this.

Reply:
oh sorry I didn't see that :-) yes that shoudl be fine